tools/manifest/sourcefile.py

```diff
@@ -30,6 +30,8 @@
 
 wd_pattern = "*.py"
 js_meta_re = re.compile(br"//\s*META:\s*(\w*)=(.*)$")
+# Matching <link rel="help" href="https://www.w3.org/TR/accelerometer/">.
```

Comment:
The `js_meta_re` pattern above is or matching META in .any.js tests, like in this file:
https://github.com/web-platform-tests/wpt/blob/master/IndexedDB/structured-clone.any.js

There's no "help" metadata as META comments in .any.js tests, but I think we'll need to support that, and extract it. But that would be in the code that uses `js_meta_re`, not as a separate regex.